packages/core-web/webpack.dev.js

```diff
@@ -47,6 +47,9 @@ module.exports = {
   serverEntry: (cb, rootFolder) => {
     const webpackServerDevConfig = merge(serverConfig, {
       mode: 'development',
+      output: {
+        path: path.join(rootFolder, 'dist'), // Change the output path based on the user defined root
```

Comment:
```suggestion
const memDistFolder = path.join(rootFolder, 'dist');
...      
        path: memDistFolder,
...
const bundleFilePath = path.join(memDistFolder, 'js/vueCommonAppFactory.min.js');
```

minor suggestion to make it clear the 2 are tied together